### PR TITLE
Fix pgettext function name typo

### PR DESCRIPTION
**Pull request by Hypolite**
This typo was aggravating me for a while. I took the opportunity to improve the documentation and the code readability.

### CODE REVIEW

File: include/pgettext.php [hunk 1]
```diff
@@ -162,25 +162,31 @@ function string_plural_select_default($n) {
...
-	if(is_array($langs) && count($langs)) {
-		if(! in_array('view/lang/en/strings.php',$langs))
-			$langs[] = 'view/lang/en/';
-		asort($langs);
-		foreach($langs as $l) {
-			$t = explode("/",$l);
-			$lang_choices[$t[2]] = $t[2];
+function get_available_languages() {
+	$langs = array();
+	$strings_file_paths = glob('view/lang/*/strings.php');
```
Comment (annando) [inline include/pgettext.php]: Could you add the ````/**/```` again? This is a little workaround for some text editors that falsely recognize the ````/*```` in the line before as comment.

Comment (Hypolite) [inline include/pgettext.php]: Some text editors? I'd be curious to know which ones.

I would be inclined to decline your request if the list doesn't include major editors (like vim, nano, emacs, eclipse, etc...).

Comment (annando) [inline include/pgettext.php]: I'm having this problem with nano.

Comment (Hypolite) [inline include/pgettext.php]: I found out that nano syntax highlighting is pretty simple. The syntax highlighting doesn't do any string tokenization, it's just simple coloring regular expressions on top of each other. Which means that nano will color according to the multiline comment regular expression even if a portion of text has already been processed by the strings regular expression.

You can somewhat fix this problem in your nano syntax highlighting config file by moving the string regular expression lower than the multiline comment regular expression. This way the comments won't be highlighted in the strings, but the string will be highlighted in the multi-line comments.

I spent some time figuring if other editors had the same issue, and vim and emacs are correctly highlighting the code above. As far as I'm concerned, it's an editor-specific issue that should not impact the code.

**Comment by Hypolite** [comment]
This really is inconvenient, it's a nano bug, I'm not comfortable with
having code workaround for an editor shortcoming. :/

**Comment by annando** [comment]
Yeah, ... I don't really like that as well. Have you tried if these changes in the config will work?

**Comment by Hypolite** [comment]
Yes, like I said instead of highlighting comments in strings, it highlights strings in comments which is less disruptive. If you need additional PHP syntax highlighting config files I can provide them since I researched the issue way more than I should have 😅

**Comment by annando** [comment]
We are talking about a config file for nano.

**Pull request merged**